### DISCUSSION TITLE
some of variables doesnt load

### DISCUSSION THREAD

**Discussion by srainbow2008** [requarks/wiki#4702] [Error / Bug Report]
Hi everyone

Is there any one who had a problem like me ?

some words arent shown correctly.

for example : 
![image](https://user-images.githubusercontent.com/94443922/141960224-b397ac3a-7f4f-4e67-bfeb-494f8307ce4d.png)
![image](https://user-images.githubusercontent.com/94443922/141960501-547fee13-e548-405a-877b-049f445379fe.png)

-----
I treid it with english language, too, and i caught the same problem.

**Comment by srainbow2008** [comment]
@ieaster1
 
 
 could you check it, please?

**Comment by ieaster1** [comment]
Your issue seems related to #1649. 

I would review and correct your locale files (Admin -> Locale -> Download Locale) then restart the wiki application.
